packages/core/src/payment/strategies/bolt/bolt-payment-strategy.ts

```diff
@@ -351,4 +356,23 @@ export default class BoltPaymentStrategy implements PaymentStrategy {
             throw new PaymentArgumentInvalidError();
         }
     }
+
+    private setExtraItemsForAnalytics() {
```

Comment:
```suggestion
    private _setExtraItemsForAnalytics() {
```

Reply:
https://github.com/bigcommerce/checkout-sdk-js/pull/1613#discussion_r980117760 😅

Reply:
@solofeed I get the private modifier above, but in TS/JS it's still not natively supported so we try and keep it consistent by adding the `_`.

Reply:
There is a new proposal to add `#` to mark properties as private and that will have better checking, but till then we wait